bitcoin/Cargo.toml

```diff
@@ -38,8 +38,9 @@ secp256k1 = { version = "0.28.0", default-features = false, features = ["hashes"
 units = { package = "bitcoin-units", version = "0.1.0", default-features = false, features = ["alloc"] }
 
 base64 = { version = "0.21.3", optional = true }
-# Only use this feature for no-std builds, otherwise use bitcoinconsensus-std.
-bitcoinconsensus = { version = "0.20.2-0.5.0", default-features = false, optional = true }
+
+# Please note, bitocinconsensus does not currently work with no-std.
+bitcoinconsensus = { version = "0.20.2-0.5.0", optional = true }
```

Comment:
This is not documenting anything and as I understand it, `bitcoinconsensus` may be able to compile on some `no_std` platforms but not all of them?